src/index.ts

```diff
@@ -1,2 +1,10 @@
-export * from './tsslib';
+import * as lib from './tsslib';
 export * from './Bridge';
+
+const tssLib = {
```

Comment:
why doing this?

Reply:
to match tss-lib interface which will be import by mpc-core-kit as custom tss-lib